modules/ROOT/pages/_partials/commons/common-document.adoc

```diff
@@ -468,6 +472,7 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 :param-tags!:
 :param-leader!:
 
+You can set expiration for a whole Collection 
```

Comment:
We do not support his feature in 3.1. The expiration can be only set to a document in 3.1.